### PR TITLE
add nameresolver as an own server

### PR DESCRIPTION
**Pull request by arnaucube**

### CODE REVIEW

File: cmd/nameserver/config.example.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+server:
+	serviceapi: 0.0.0.0:8000
```
Comment (ed255) [inline cmd/nameserver/config.example.yaml]: Please, use 2 space indentation like in the rest of the file.

File: cmd/relay/endpoint/name.go [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 import (
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/gin-gonic/gin"
+	cfg "github.com/iden3/go-iden3/cmd/relay/config"
 	common3 "github.com/iden3/go-iden3/common"
 	"github.com/iden3/go-iden3/services/namesrv"
 )
 
 func handleVinculateId(c *gin.Context) {
```
Comment (ed255) [inline cmd/relay/endpoint/name.go]: shouldn't this handler be removed from here?

File: cmd/relay/config/load.go [hunk 3]
```diff
@@ -156,8 +156,8 @@ func LoadClaimService(mt *merkletree.MerkleTree, rootservice rootsrv.Service, ks
 	return claimsrv.New(mt, rootservice, signsrv.New(ks, acc))
 }
 
-func LoadNameService(identityservice identitysrv.Service, claimservice claimsrv.Service, ks *keystore.KeyStore, acc accounts.Account, domain string, namespace string) namesrv.Service {
-	return namesrv.New(claimservice, identityservice, signsrv.New(ks, acc), domain)
+func LoadNameService(claimservice claimsrv.Service, ks *keystore.KeyStore, acc accounts.Account, domain string, namespace string) namesrv.Service {
```
Comment (ed255) [inline cmd/relay/config/load.go]: Why is the relay loading the name service?

File: cmd/relay/commands/server.go [hunk 4]
```diff
@@ -48,7 +48,7 @@ func cmdStart(c *cli.Context) error {
 	rootservice := cfg.LoadRootsService(client)
 	claimservice := cfg.LoadClaimService(mt, rootservice, ks, acc)
 	idservice := cfg.LoadIdService(client, claimservice, storage)
-	nameservice := cfg.LoadNameService(idservice, claimservice, ks, acc, cfg.C.Domain, cfg.C.Namespace)
+	nameservice := cfg.LoadNameService(claimservice, ks, acc, cfg.C.Domain, cfg.C.Namespace)
```
Comment (ed255) [inline cmd/relay/commands/server.go]: Why is the relay loading the config for a name service?

File: cmd/nameserver/config.example.yaml (hunk 1 above)
Comment (arnaucube) [reply cmd/nameserver/config.example.yaml]: done

File: cmd/relay/endpoint/name.go (hunk 2 above)
Comment (arnaucube) [reply cmd/relay/endpoint/name.go]: done, now is only in nameserver, not more in relay

File: cmd/relay/config/load.go (hunk 3 above)
Comment (arnaucube) [reply cmd/relay/config/load.go]: right, removed

File: cmd/relay/commands/server.go (hunk 4 above)
Comment (arnaucube) [reply cmd/relay/commands/server.go]: right, removed

**Pull request merged**